src/main/kotlin/com/demonwav/mcdev/platform/mcp/actions/CopyATAction.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Minecraft Dev for IntelliJ
+ *
+ * https://minecraftdev.org
+ *
+ * Copyright (c) 2017 minecraft-dev
+ *
+ * MIT License
+ */
+
+package com.demonwav.mcdev.platform.mcp.actions
+
+import com.demonwav.mcdev.platform.mcp.McpModuleType
+import com.demonwav.mcdev.platform.mixin.util.findFirstShadowTarget
+import com.demonwav.mcdev.util.getDataFromActionEvent
+import com.demonwav.mcdev.util.invokeLater
+import com.intellij.openapi.actionSystem.AnAction
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.DataKeys
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.ui.popup.Balloon
+import com.intellij.openapi.ui.popup.JBPopupFactory
+import com.intellij.openapi.wm.WindowManager
+import com.intellij.psi.*
+import com.intellij.ui.LightColors
+import com.intellij.ui.awt.RelativePoint
+import java.awt.Toolkit
+import java.awt.datatransfer.StringSelection
+
+class CopyATAction : AnAction() {
```

Comment:
I know it's weird, but the way we name stuff this should be `CopyAtAction`.

Reply:
Thanks for catching stuff. It's not weird though, it has it's reasons 😛.

Reply:
Well I left it like that since it's an acronym :P

Would `CopyAccessTransformerAction` be better?

Reply:
No. `CopyAtAction` is correct. [The reasoning is explained in the style guide.](https://www.demonwav.com/style#s5.3-camel-case)

Reply:
I get that it's correct, but using the non abbreviated one potentially makes us both happy :p

Reply:
I see. No, please call it `CopyAtAction`. Within the context of this code it is clear what that means, no need to write out the `AccessTransformers` bit. I think writing it out looks worse.